tests/e2e/clipboard.spec.ts

```diff
@@ -74,14 +75,12 @@ test("copy to clipboard", async ({ page, browserName, context }) => {
     );
   }
 
+  // TODO: Navigating to radicle.local with a non default port doesn't work currently
```

Comment:
Are you going to follow up on this in this PR? If not, I'd remove it, I think we're already aware of this shortcoming.